how2meet/db/models.py

```diff
@@ -18,15 +18,14 @@
 class Event(Base):
     __tablename__ = "events"
 
-    id = Column(String(36), primary_key=True, index=True)
+    id = Column(Uuid, primary_key=True, index=True)
+    auth_token = Column(Uuid, index=True)
     name = Column(String(150))
     created = Column(DateTime)
     start_time = Column(DateTime)
     end_time = Column(DateTime)
     all_day = Column(Boolean)
     location = Column(String(150))
-    organizer_name = Column(String(50))
```

Comment:
I get that you don't want user+pass combo for editing events and I'm cool with that, but having a field to identify who (or what, if its an organization) is planning the event seems like it could be useful.

Reply:
Yeah, I'll bring that back. Honestly was an over sight to completely strip out the organizer name.